### ISSUE TITLE
Adding new / own commands (not command stickys)

### ISSUE THREAD

**Issue by Dvd15** [proophboard/cody-engine#57]
Hi, David Werner, from Team MangoQuadrat here, we are doing our project semester with BitExpert right now and were wondering if someone can explain how to add your own commands, to use, in the Proophboard Tab. We tried to trace the "Hey ${user}, Cody here. Before we can start..."-output back to the "/connect ..."-command but so far we could only find the pbTab window and an addEventListener on said window but not yet where "msg" gets filled with the needed information. 

_This code can be found in (packages/play/src/infrastructure/cody/cody-message-server.ts)_
![Screenshot from 2024-04-20 18-58-46](https://github.com/proophboard/cody-engine/assets/57320627/d655cdd3-adfb-426a-ade5-b077d8a07c73)

We also couldn't find if there are any other commands besides /connect and /help, and if so, where they are implemented or even declared.

Tl/dr we would like to know where the commands "/connect" and "/help" are implemented and declared

**Comment by codeliner** [comment]
Closing this, as you found a good way to deal with the issue ;)
